cmd/ro/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/cloudflare/redoctober/client"
+	"github.com/cloudflare/redoctober/core"
+
+	"github.com/lziest/gopass"
+)
+
+var action, user, pswd, userEnv, pswdEnv, server, caPath string
+
+var owners, lefters, righters, inPath, labels, outPath, outEnv string
+
+var uses int
+
+var time, users string
+
+func registerFlags() {
+	flag.StringVar(&action, "action", "", "client action")
```

Comment:
Maybe having the action as the first (non-flag) argument would make this easier to use?